docs/products/kafka/howto/viewing-resetting-offset.rst

```diff
@@ -0,0 +1,145 @@
+
+Define the configuration file
+-----------------------------
+
+#. Create the Java keystore and truststore for your Aiven for Apache Kafka service using the :ref:`dedicated Aiven CLI command <avn_service_user_kafka_java_creds>`.
+
+#. Create a ``consumer.properties`` properties file pointing to the keystore and truststore files with the following entries:
+
+* ``security.protocol``: security protocol, SSL for the default TLS security settings
+* ``ssl.keystore.type``: keystore type, ``PKCS12`` for the keystore generated with the :ref:`dedicated Aiven CLI command <avn_service_user_kafka_java_creds>`
+* ``ssl.keystore.location``: keystore location on the file system
+* ``ssl.keystore.password``: keystore password
+* ``ssl.truststore.location``: truststore location on the file system
+* ``ssl.truststore.password``: truststore password
+* ``ssl.key.password``: keystore password
+
+.. Tip::
+
+    The ``avn service user-kafka-java-creds`` :ref:`Aiven CLI command <avn_service_user_kafka_java_creds>` accepts a ``--password`` parameter setting the same password for the truststore, keystore and key
+   
+An example of the ``consumer.properties`` content is the following::
+
+    security.protocol=SSL
+    ssl.keystore.type=PKCS12
+    ssl.keystore.location=client.keystore.p12
+    ssl.keystore.password=changeit
+    ssl.key.password=changeit
+    ssl.truststore.location=client.truststore.jks
+    ssl.truststore.password=changeit
+
+
+Managing consumer group offsets with ``kafka-consumer-groups.sh``
+-----------------------------------------------------------------
+
+The ``kafka-consumer-groups.sh`` tool enables to manage consumer group offsets, the following commands are available.
+
+List the active consumer groups
+'''''''''''''''''''''''''''''''''''''''''
+
+To list the currently active consumer groups use the following command replacing the ``demo-kafka.my-project.aivencloud.com:17072`` with the Aiven for Apache Kafka service URI:
+
+::
+
+    kafka-consumer-groups.sh \
+        --bootstrap-server demo-kafka.my-project.aivencloud.com:17072 \
+        --command-config consumer.properties \
+        --list
+
+An example of the ``list`` option, showing two consumer groups, is the following:
+
+.. code:: text
+
+    my-group
+    my-group-2
+
+Retrieve the details of a consumer group
+''''''''''''''''''''''''''''''''''''''''''''''''
+
+To retrieve the details of a consumer group use the following command replacing the ``demo-kafka.my-project.aivencloud.com:17072`` with the Aiven for Apache Kafka service URI and the ``my-group`` with the required consumer group name:
+
+::
+
+    kafka-consumer-groups.sh \
+        --bootstrap-server demo-kafka.my-project.aivencloud.com:17072 \
+        --command-config consumer.properties \
+        --group my-group \
+        --describe
+          
+An example of the ``describe`` option, showing the details of the ``my-group`` consumer groups, is the following:
```

Comment:
I'd rephrase "An example of the ``describe`` option, showing the details of the ``my-group`` consumer groups, is the following:" to use more simple language "The details of the consumer group will be printed out in the output"